assets/tests/components/detours/diversionPage.saveDetour.test.tsx

```diff
@@ -40,7 +40,7 @@ beforeEach(() => {
   jest.mocked(getTestGroups).mockReturnValue([])
   jest
     .mocked(fetchNearestIntersection)
-    .mockResolvedValue("Returned Intersection")
+    .mockReturnValue(neverPromise())
```

Comment:
The intersection isn't used in these tests and it's just easier to have the promise never return, because otherwise you run into "An update inside a test was not wrapped in act(...)."